### ISSUE TITLE
is `initialVisibleMonth` now required for DayPickerRangeController?

### ISSUE THREAD

**Issue by terebentina** [airbnb/react-dates#613]
I am calling `DayPickerRangeController` like this:
```
              <DayPickerRangeController
                startDate={this.state.customIntervalStart}
                endDate={this.state.customIntervalEnd}
                onDatesChange={this.onCustomDatesChange}
                focusedInput={this.state.focusedInput}
                onFocusChange={this.onDatePickerFocusChange}
                minimumNights={0}
                keepOpenOnDateSelect={false}
                renderCalendarInfo={() => (
                  <CalendarInfo onClick={this.onClearDatesClick} className={styles.calendarInfo} />
                )}
              />
```
It used to be fine until react-dates 12.2.0. With every 12.2 so far I am now getting
```
Uncaught TypeError: props.initialVisibleMonth is not a function
    at new DayPicker (DayPicker.js:555)
```

Am I supposed to specify `initialVisibleMonth` in `DayPickerRangeController` now? The docs still say it's optional.

**Comment by Si1kIfY** [comment]
Same issue here but slightly different error message:

```
Uncaught (in promise) TypeError: props.initialVisibleMonth is not a function
    at new DayPicker (DayPicker.js:555)
    at ReactCompositeComponent.js:294
    at measureLifeCyclePerf (ReactCompositeComponent.js:75)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (ReactCompositeComponent.js:293)
    at ReactCompositeComponentWrapper._constructComponent (ReactCompositeComponent.js:279)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:187)
    at Object.mountComponent (ReactReconciler.js:45)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:370)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:257)
    at Object.mountComponent (ReactReconciler.js:45)
```

**Comment by aleib** [comment]
Same issue. As mentioned by @terebentina specifying the initialVisibleMonth removes the error.
So something like:
```initialVisibleMonth={() => moment().subtract(1, 'month')}```

**Comment by majapw** [comment]
Thanks for the catch! I commented on the PR. :)

**Comment by majapw** [comment]
Fixed in v12.2.4!
